x/epochs/spec/README.md

```diff
@@ -0,0 +1,39 @@
+<!--
+order: 0
+title: "Epochs Overview"
+parent:
+  title: "epochs"
+-->
+
+# `epochs`
+
+## Abstract
+
+The purpose of `epochs` module is to provide generalized epoch interface to other modules so that they can easily implement epochs without keeping own code for epochs.
+
+## Implementation
+
+### Keeper functions
+```go
+// Keeper is the interface for lockup module keeper
+type Keeper interface {
+	// CreateEpochCounter Returns full balance of the module
+  // All of these epoch counters could be set on epoch module's genesis
+  // if startTime is not set, we could use genesisTime (ctx.BlockTime) at the time of InitChain
+	CreateEpochCounter(ctx sdk.Context, epochIdentifier string, duration time.Duration, startTime time.Time)
```

Comment:
CreateEpochCounter does not exist on this implementation. Is this intended to be SetEpochInfo?

Reply:
It's initial spec and for more clear spec, there will be following PR